web/pgadmin/tools/psql/__init__.py

```diff
@@ -404,33 +416,32 @@ def get_conn_str_win(manager, db):
             manager.port,
             db if db != '' else 'postgres',
             underscore_unescape(manager.user) if manager.user else 'postgres',
-            manager.ssl_mode,
-            True if manager.sslcompression else False,
+            sslmode,
+            sslcompression,
         )
 
-    if manager.hostaddr:
-        conn_attr = " {0} hostaddr='{1}'".format(conn_attr, manager.hostaddr)
+    if hostaddr:
+        conn_attr = " {0} hostaddr='{1}'".format(conn_attr, hostaddr)
 
-    if manager.passfile:
-        conn_attr = " {0} passfile='{1}'".format(conn_attr,
-                                                 get_complete_file_path(
-                                                     manager.passfile))
+    if passfile:
+        conn_attr = " {0} passfile='{1}'".format(
+            conn_attr, get_complete_file_path(passfile))
 
-    if get_complete_file_path(manager.sslcert):
+    if get_complete_file_path(sslcert):
         conn_attr = " {0} sslcert='{1}'".format(
-            conn_attr, get_complete_file_path(manager.sslcert))
+            conn_attr, get_complete_file_path(sslcert))
 
-    if get_complete_file_path(manager.sslkey):
+    if get_complete_file_path(sslkey):
         conn_attr = " {0} sslkey='{1}'".format(
-            conn_attr, get_complete_file_path(manager.sslkey))
+            conn_attr, get_complete_file_path(sslkey))
 
-    if get_complete_file_path(manager.sslrootcert):
+    if get_complete_file_path(sslrootcert):
         conn_attr = " {0} sslrootcert='{1}'".format(
-            conn_attr, get_complete_file_path(manager.sslrootcert))
+            conn_attr, get_complete_file_path(sslrootcert))
 
-    if get_complete_file_path(manager.sslcrl):
+    if get_complete_file_path(sslcrl):
         conn_attr = " {0} sslcrl='{1}'".format(
-            conn_attr, get_complete_file_path(manager.sslcrl))
+            conn_attr, get_complete_file_path(sslcrl))
 
```

Comment:
We don't pass all the params to psql like we do pgAdmin psycopg2 connect? Shouldn't we use `create_connection_string` here as well?

Reply:
Fixed. Call create_connection_string